### ISSUE TITLE
Browser opens twice

### ISSUE THREAD

**Issue by manobalas** [chrome-php/headless-chromium-php#223]
I'm using macbook bigsur, 
when i call the code, it opens the browser 2 times and same evaluation happening 2 times.
Can any one help to solve this

**Comment by enricodias** [comment]
I couldn't reproduce that behavior here.

I just ran the first example of the readme in 2 macbooks, one with Big Sur and the other with Mojave. In both cases it worked as expected, only one chrome instance was opened.

**Comment by manobalas** [comment]
can we connect on skype(live:manobala.s)?
so i can show you ...

**Comment by enricodias** [comment]
That wouldn't help. If the error is not reproducible in another machine, the issue is probably in your end. What chrome version are you using and what is the specific code that causes this behavior?

**Comment by manobalas** [comment]
Here is the code that I've used

```
<?php
require 'vendor/autoload.php';
use HeadlessChromium\BrowserFactory;
$browserFactory = new BrowserFactory();
$browser = $browserFactory->createBrowser(['headless' => false]);

try {
    $page = $browser->createPage();
    $page->navigate('https://api.upstox.com/index/login')->waitForNavigation();
    $page->evaluate("document.getElementsByClassName('pure-input-1 input-fields')[0].value = 918");
    $page->evaluate("document.getElementsByClassName('pure-input-1 input-fields')[1].value = ''");
    $page->evaluate("document.getElementsByClassName('pure-input-1 input-fields')[2].value = ''");
    $page->evaluate("document.getElementsByClassName('pure-button-primary')[0].click()");
    $page->evaluate("document.getElementsByClassName('pure-button-primary')[0].click()");
} finally {
    $browser->close();
}
```

**Comment by manobalas** [comment]
Version 90.0.4430.212 (Official Build) (x86_64)
Updated now....
Even now same issue...
Version 91.0.4472.77 (Official Build) (x86_64)

**Comment by enricodias** [comment]
I tested the same code here and it worked normally.

**Comment by manobalas** [comment]
anything fishy ?

When i run that, it completes all the commands and then again it runs again... 
Another Question: I;m planning to put this code in ubuntu server, is there the problem will be ? or it will be good ?

**Comment by enricodias** [comment]
Here it just runs all commands once and exit. If you plan to use it in a server, test in a server.

**Comment by GrahamCampbell** [comment]
I think this is possibly fixed in v0.11.0 owing to the removal of the version checking code. Can you please confirm?
